code/lib/store/template/stories/globals.stories.ts

```diff
@@ -29,6 +29,7 @@ export const Events = {
   ],
   play: async ({ canvasElement }: PlayFunctionContext<any>) => {
     const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
+    await channel.emit('updateGlobals', { globals: { foo: 'fooValue' } });
```

Comment:
Why is this added?

Reply:
the same raison as previous the discussed one time with @JReinhold  i think we need to place the reset state in the begining of the play function ,because if in case your play function fails in the middle will never reach the end to reset the state, so if you navigate back to this story you will not pass unless you do it manually.

i think it safer to the state in the begining of the play function